.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+**Merge checklist:**
+- [ ] Please do not import models directly in course discovery instead follow the approach used in “get_whitelisted_course_skills” 
```

Comment:
**Suggestion:** 

_"instead follow the approach used in “get_whitelisted_course_skills”"_
 -> 
_"e:g if you want to import CourseSkills in Discovery, use the utility get_whitelisted_course_skills instead of directly importing it."_